source/html/daily.html

```diff
@@ -20,8 +20,7 @@
       <div id="journal-content">
         <div id="title">
           <div class="date">
-            <!-- TODO: this needs to fetch today's date-->
-            <h1 class="date">May 24th</h1>
+            <h1 class="date"></h1>
```

Comment:
For some reason, on my branch, the date is never fetched. Does that happen to you?

Reply:
@yuzilyu I was pair programming with Brett earlier; worked fine for us on his browser. Maybe add it as an issue?

Reply:
Hmmm, like Noah said it worked fine on my browser, may need to look into this

Reply:
I think we should have e2e test cases for this

Reply:
on my browser, it either displays nothing or 5/24/2021